app/src/org/commcare/views/widgets/GridWidget.java

```diff
@@ -78,7 +79,8 @@ public GridWidget(Context context, FormEntryPrompt prompt,
         // The max width of an icon in a given column. Used to line
```

Comment:
Ok then separate comment for the height maybe? At a high level they're both used to configure the GridView appearance, right?